kong.conf.default

```diff
@@ -452,17 +452,32 @@
                                  # To read the file again after modifying it,
                                  # Kong must be reloaded.
 
-#dns_order = LAST,SRV,A,CNAME    # the order in which to resolve different
+#dns_order = LAST,SRV,A,CNAME    # The order in which to resolve different
                                  # record types. The `LAST` type means the
                                  # type of the last successful lookup (for the
                                  # specified name). The format is a (case
                                  # insensitive) comma separated list.
 
+#dns_stale_ttl = 4.0             # Once a record expires, how much longer (in
```

Comment:
When a value's unit of time is in seconds, I think it makes it simpler for the user if we avoid the fractional part in the defaults. It makes things unnecessary complicated.